internal/handlers/pg/pgdb/pool.go

```diff
@@ -413,3 +462,23 @@ func (pgPool *Pool) SchemaStats(ctx context.Context, schema, collection string)
 	}
 	return &res, nil
 }
+
+// schemaExists returns true if given schema exists.
+func (pgPool *Pool) schemaExists(ctx context.Context, db string) (bool, error) {
```

Comment:
I constantly wonder, how the db relates to schema (db contains schema in pg, and schema relates to mongodb collection names, but what is table then?).

And now we have yet another entity - settings table.

Let's call clarify our scheme in the comments and write a proper documentation for it 
* what are the fields inside that settings table
* why the structure is like this
* why we need another one entity.
* what else important
* etc.

The documentation can be added i.e. in the settings.go file, where the Settings Table term introduced.

Reply:
I've added a comment to `CreateSettingsTable`. I think it's enough for now. We can't clarify anything in that PR, as it requires refactoring to `pgPool` and handlers methods.